mi/dataset/driver/ctdav_n/auv/ctdav_n_auv_recovered_driver.py

```diff
@@ -9,6 +9,8 @@
 Release notes:
 
 Initial Release
+
```

Comment:
Please see https://stackoverflow.com/questions/2536307/how-do-i-deprecate-python-functions

Reply:
Installed and imported the "deprecated" python package and deprecated the telemetered and recovered drivers.